src/Noobot.Tests.Unit/packages.config

```diff
@@ -70,4 +68,12 @@
   <package id="System.Xml.ReaderWriter" version="4.3.0" targetFramework="net461" />
   <package id="System.Xml.XDocument" version="4.3.0" targetFramework="net461" />
   <package id="WebsocketClientLite.PCL" version="4.0.6" targetFramework="net461" />
+  <package id="xunit" version="2.2.0" targetFramework="net461" />
+  <package id="xunit.abstractions" version="2.0.1" targetFramework="net461" />
+  <package id="xunit.assert" version="2.2.0" targetFramework="net461" />
+  <package id="xunit.core" version="2.2.0" targetFramework="net461" />
+  <package id="xunit.extensibility.core" version="2.2.0" targetFramework="net461" />
+  <package id="xunit.extensibility.execution" version="2.2.0" targetFramework="net461" />
+  <package id="xunit.runner.visualstudio" version="2.2.0" targetFramework="net461" developmentDependency="true" />
+  <package id="xunit.should" version="1.1" targetFramework="net461" />
```

Comment:
Careful using this version. I had problems as it uses an older xunit. Can this be replaced with `Xunit.Should.Extension`